### PR TITLE
 fix!: replace positional use with explicity --name flag

### PR DESCRIPTION
**Pull request by wtrocki**
Context and discussion #927 
## Verification

rhoas kafka use (will print help)
rhoas kafka use --name=yourkafka
rhoas kafka use --id=id

### CODE REVIEW

**Comment by wtrocki** [comment]
@craicoverflow as discussed this is something that we can employ to minimize confusion. Let me know if that works for you.

File: pkg/cmd/kafka/use/use.go [hunk 1]
```diff
@@ -70,7 +68,7 @@ func NewUseCommand(f *factory.Factory) *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&opts.id, "id", "", opts.localizer.MustLocalize("kafka.use.flag.id"))
-
+	cmd.Flags().StringVar(&opts.id, "name", "", opts.localizer.MustLocalize("kafka.use.flag.name"))
```
Comment (rkpattnaik780) [inline pkg/cmd/kafka/use/use.go:71]:
```suggestion
	cmd.Flags().StringVar(&opts.name, "name", "", opts.localizer.MustLocalize("kafka.use.flag.name"))
```

File: pkg/cmd/registry/use/use.go [hunk 2]
```diff
@@ -63,6 +61,7 @@ func NewUseCommand(f *factory.Factory) *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&opts.id, "id", "", opts.localizer.MustLocalize("registry.use.flag.id"))
+	cmd.Flags().StringVar(&opts.id, "name", "", opts.localizer.MustLocalize("registry.use.flag.name"))
```
Comment (rkpattnaik780) [inline pkg/cmd/registry/use/use.go:64]:
```suggestion
	cmd.Flags().StringVar(&opts.name, "name", "", opts.localizer.MustLocalize("registry.use.flag.name"))
```

**Review by rkpattnaik780 — CHANGES_REQUESTED** [review]

**Comment by wtrocki** [comment]
@rkpattnaik780 Forgot to push local change. Thanks for spotting problem. This is good now

**Review by rkpattnaik780 — APPROVED** [review]

File: pkg/localize/locales/en/cmd/registry_crud.en.toml [hunk 3]
```diff
@@ -87,6 +87,9 @@ one = 'Use a Service Registry instance'
 [registry.cmd.use.longDescription]
 one = ''' 
 Select a Service Registry instance to use with all instance-specific commands.
+You can specify a Service Registry instance by --name or --id.
+
+When you set the Service Registry instance to be used, it is set as the current instance for all rhoas service-registry artifacts commands.
```
Comment (wtrocki) [inline pkg/localize/locales/en/cmd/registry_crud.en.toml:92]:
```suggestion
When you set the Service Registry instance to be used, it is set as the current instance for all rhoas service-registry artifact commands.
```

**Pull request merged**